src/sentry/api/endpoints/group_integration_details.py

```diff
@@ -4,17 +4,34 @@
 
 from rest_framework.response import Response
 
-from sentry import analytics
+from sentry import analytics, features
 from sentry.api.bases import GroupEndpoint
 from sentry.api.serializers import serialize
 from sentry.api.serializers.models.integration import IntegrationIssueConfigSerializer
 from sentry.integrations import IntegrationFeatures
 from sentry.integrations.exceptions import IntegrationError, IntegrationFormError
 from sentry.models import ExternalIssue, GroupLink, Integration
 
+MISSING_FEATURE_MESSAGE = 'Your organization does not have access to this feature.'
+
 
 class GroupIntegrationDetailsEndpoint(GroupEndpoint):
+    def _has_issue_feature(self, organization, user):
+        has_issue_basic = features.has('organizations:integration:issue_basic',
+                                       organization,
+                                       actor=user)
+
+        has_issue_sync = features.has('organizations:integration:issue_sync',
+                                      organization,
+                                      actor=user)
+
+        return has_issue_sync or has_issue_basic
```

Comment:
You have this function in two places. Could it be a method of the User or Organization instead?

Reply:
IMO I don't think we need to try to deduplicate this, since it's just a couple of lines

Reply:
yeah we generally haven't moved feature checks into methods on user / org and if we did i think it would make those models very cluttered. i could subclass `GroupEndpoint` but i don't know that that would make this much cleaner

Reply:
Fair points :+1: